src/templates/ratings/rating.html

```diff
@@ -7,7 +7,7 @@
   {% defer (url=api('review', [id])) %}
     <div class="reviews reviews-listing single-review">
       <ul class="ratings-placeholder-inner">
-        {{ rating(this) }}
+        {{ review(this) }}
```

Comment:
We had switched to "ratings" from "reviews" at one point but I don't remember the full story behind it. Doesn't really matter I agree with keeping it consistent and I noticed you changed the file names too which is great.